models/Update.model.ts

```diff
@@ -0,0 +1,8 @@
+import { User } from "./User.model";
+
+export type Update = {
```

Comment:
The name of this type doesn't provide a clear description. What does it really represent?

Reply:
Changed to "Comment"